_includes/license-overview.html

```diff
@@ -43,7 +43,7 @@ <h3 class="license-family-name">
     {% endif %}
 
     {% for variation in variations %}
-    <div {% if variation.tab-slug %}id="{{ variation.tab-slug }}" {% endif %}class="variation-tab {% if variation.tab-slug %}variation-{{ variation.tab-slug }}{% endif %} js-variation-tab{% if license.tab-slug == variation.tab-slug %} active{% endif %}">
+    <div {% if variation.id %}id="{{ variation.id | remove: '.' | remove: '/licenses/' }}" {% endif %}class="variation-tab {% if variation.id %}variation-{{ variation.id | remove: '.' | remove: '/licenses/' }}{% endif %} js-variation-tab{% if license.id == variation.id %} active{% endif %}">
```

Comment:
I believe `{{ variation.id | remove: '/licenses/' | slugify }}` should achieve roughly the same ends by replacing all non-alphanumeric characters (and replace `.`s with `-`s to somewhat preserve existing behavior).